.stylelintrc

```diff
@@ -16,8 +17,10 @@
         ]
       }
     ],
+    "color-function-notation": null,
```

Comment:
Latest stylelint-config-standard wants us to write

```css
rgba(1 2 3 0.5)
```

instead of

```css
rgba(1, 2, 3, 0.5)
```

Didn't seem worth worrying about too much, so I turned off the rule. There's only 1 violation, though, so it would be easy to turn back on if we want.